package.json

```diff
@@ -4,8 +4,13 @@
   "description": "API module for the Screwdriver CD service",
   "main": "index.js",
   "scripts": {
-    "pretest": "jscs .",
```

Comment:
The linting upgrade is a big change on its own, so we'll make a separate PR for it